.github/workflows/windows_noetic_build.yml

```diff
@@ -0,0 +1,49 @@
+name: Windows-Noetic-Build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  schedule:
+    - cron: '0 5 * * *'
+
+jobs:
+  windows_ci:
+    name: Noetic
+    runs-on: windows-2019
+    env:
+      ROS_DISTRO: noetic
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        path: src/plugin_loader
+    - name: Build
+      shell: cmd
+      run: |
+        choco sources add -n=roswin -s https://aka.ms/ros/public --priority 1
+        choco install ros-%ROS_DISTRO%-desktop_full -y --no-progress
```

Comment:
We should figure out how to not install all of ROS just to build this package. I started doing this with [descartes](https://github.com/swri-robotics/descartes_light/pull/90) but haven't been able to figure out how to get Windows to find the installed packages without source `/opt/ros/.../setup.bash`

Reply:
I agree, I see if I can figure it out.